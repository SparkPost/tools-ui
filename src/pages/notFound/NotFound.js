import React from 'react';

import './NotFound.scss';

const NotFound = () => (
  <div className='flex center-xs'>
    <div className='col-xs-12 notFound'>
      <h5 className='notFound__desc'>The page you are looking for can't be found.</h5>
      <h1 className='notFound__code'>404</h1>
    </div>
  </div>
);

export default NotFound;
