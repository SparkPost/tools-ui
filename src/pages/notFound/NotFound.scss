@import "~@sparkpost/styles/src/assets/scss/shared/config";
@import "~@sparkpost/styles/src/assets/scss/shared/functions";

.notFound {
  padding: rem(42) 0;

  &__desc {
    color: grayscale(4);
    font-family: $gotham-regular;
  }

  &__code {
    padding: rem(39) 0;
    font-size: rem(90);
  }
}
