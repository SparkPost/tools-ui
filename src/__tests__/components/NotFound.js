import React from 'react';
import renderer from 'react-test-renderer';
import NotFound from 'pages/notFound/NotFound';

describe('NotFound', () => {
  test('should render 404', () => {
    // Probably want to mock react-router somehow
    expect(renderer.create(<NotFound/>)).toMatchSnapshot();
  });
});
