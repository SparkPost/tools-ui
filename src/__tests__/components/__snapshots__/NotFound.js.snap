// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NotFound should render 404 1`] = `
<div
  className="flex center-xs"
>
  <div
    className="col-xs-12 notFound"
  >
    <h5
      className="notFound__desc"
    >
      The page you are looking for can't be found.
    </h5>
    <h1
      className="notFound__code"
    >
      404
    </h1>
  </div>
</div>
`;
